src/ItemValue.ts

```diff
@@ -132,7 +132,7 @@ export class ItemValue implements IItemValue {
                 this.update();
             } else {
                 // Otherwise save the new version to memory
-                this.updateLocalStorage();
+                this.updateLocalStorage(true);
```

Comment:
If nothing is passed in (or autoSave isn't enabled), the value will not save to localStorage